guide/getting-started/hardware.md

```diff
@@ -86,13 +88,13 @@ Thousands of computers participate in the Bitcoin network. They are called nodes
    height = 400
 %}
 
-Dedicated node hardware is particularly relevant for heavy users of the Lightning network. While a bitcoin node can be turned off and on and can easily catch up with network activity, Lightning nodes need to be online at all times to maintain payment channels with other nodes. Smartphones are typically online 24/7 but are not suited well for this task due to large amounts of data transfer and storage capacity requirements.
+Dedicated node hardware is particularly relevant for heavy users of the Lightning network. While a bitcoin node can be turned off and on, Lightning nodes need to be online at all times to maintain payment channels with other nodes. Smartphones are typically online 24/7 but are not suited for this task due to large amounts of data transfer and storage capacity requirements.
```

Comment:
```suggestion
Dedicated node hardware is particularly relevant for users of the Lightning network. While a bitcoin node can be turned off and on, Lightning nodes need to be online at all times to maintain payment channels with other nodes. Smartphones are typically online 24/7 but are not suited for this task due to large amounts of data transfer and storage capacity requirements.
```

We should add an entry for Lightning nodes to this page in a separate PR and call this section a Bitcoin Node. There is a lot of nuances here to consider, for example you can run a Lightning node and use something like Neutrino to retrieve blockchain data.

Reply:
LN is out of scope of this PR. Just doing minor changes here.

Reply:
I'd just keep it simple and link to the [Glossary node page](https://bitcoin.design/guide/glossary/node/), and cover all the Bitcoin/Lightning node details there.

Reply:
Done.
Added a bunch of links to that awesome page https://github.com/BitcoinDesign/Guide/pull/438/commits/6a36ce616e679c1b4925d434411b8d3d550b29fd